solvers/glum.py

```diff
@@ -25,11 +28,12 @@ def set_objective(self, X, y, lmbd, fit_intercept):
             alpha=self.lmbd / n_samples,
             fit_intercept=fit_intercept,
             gradient_tol=1e-8,
+            max_iter=1
         )
         warnings.filterwarnings('ignore', category=ConvergenceWarning)
 
-    def run(self, n_iter):
-        self.model.max_iter = n_iter + 1
+    def run(self, tol):
+        self.model.gradient_tol = tol ** 2
```

Comment:
I am not sure this is the right scaling.
I would say we should remove the `**2` and scale this with the norm of the gradient in 0?